database/migrations/default/1758732024749_ems__incidents_add_is_deleted/down.sql

```diff
@@ -0,0 +1,3 @@
+alter table "public"."ems__incidents" drop column "is_deleted";
+
+drop index ems__incidents_is_deleted_index;
```

Comment:
~I'm seeing an error when running this down migration, and I think this index is dropped with the column.~ you fixed it! 🙏🙏